plugins/community/mfridman-go-json/source.yaml

```diff
@@ -0,0 +1,4 @@
+source:
+  github:
+    owner: mfridman
```

Comment:
Hmm... not sure about this maintainer. 😀

Reply:
😰